.github/workflows/Tests.yml

```diff
@@ -72,16 +72,37 @@ jobs:
         with:
           version: '${{ matrix.version }}'
           arch: ${{ matrix.arch }}
-      - uses: actions/cache@v4
+      - uses: julia-actions/cache@v1
+      - uses: julia-actions/julia-buildpkg@v1
+      # TODO: Use julia-actions/julia-runtest when test_args are supported
```

Comment:
I think it's a quite unfortunate consequence of https://github.com/TuringLang/Turing.jl/pull/2237 that we have to work around the standard tooling in the Julia ecosystem.